### DISCUSSION TITLE
User Annotations

### DISCUSSION THREAD

**Discussion by malteos** [alephdata/aleph#1518] [Ideas]
Hi all,

I'm currently working on the integration of an annotation tool into Aleph. Users should be able to manually create labels for ML-based data analysis (e.g., text classification, NER, ...). 

**Resources**
- Inspiration from Spacy's Prodigy: https://prodi.gy/demo
- React Text Annotation: https://mcamac.github.io/react-text-annotate/

**Screenshot (WIP)**

<img width="724" alt="screenshot" src="https://user-images.githubusercontent.com/7977470/80961205-bb374480-8e0a-11ea-95ba-a3080578e4bb.png">

**Open questions**
- How to storage user data? In ES or DB?
- Feedback loop to ML-models

Feedback and help is very much welcome!

**Comment by malteos** [comment]
Current progress:

![Screenshot 2020-09-03 at 10 43 21](https://user-images.githubusercontent.com/7977470/92092818-dc1ca000-edd2-11ea-9c09-8f6df6caaa17.png)

See https://github.com/malteos/qaleph/tree/qaleph/ui/src/qaleph/components/Annotations

**Comment by Rosencrantz** [comment]
@pudo @sunu Thoughts on this...
